### ISSUE TITLE
Run Time Is Off

### ISSUE THREAD

**Issue by bfad** [bfad/lspec#1]
The reported time it takes is too long. I used the unix `time` command to time the lspec command test suite when it only had one test, and this is the output:

```
$> time lasso9 spec/command/lspec.spec.inc
.

Finished in 1.428784 seconds
1 test, 0 failures

real    0m1.228s
user    0m1.213s
sys     0m0.041s
```

As you can see, the real time is shorter than the run time!

**Comment by bfad** [comment]
It looks like this is due to nested test groups. Each test group is adding its runtime to the time; however, this means that the time it takes to run the nested test groups is actually being counted twice since it is being counted with the top-level test group.

**Comment by bfad** [comment]
Fixed in 5ebd25231ba98cb3624f223cc7d7d4a6fbc2e2a2
